go.mod

```diff
@@ -1,24 +1,21 @@
 module github.com/vmware/govmomi
 
-go 1.22
+go 1.21 // required for slices/maps package and built-in clear function
```

Comment:
I had bumped to 1.22 here just because we support the latest 2 Go minor versions and only test w/ those versions.

Reply:
Hmm, how do you want to handle this? Should we consider changing this behavior to "leave version alone unless the library needs a newer version?"